packages/apputils-extension/src/notificationplugin.tsx

```diff
@@ -678,15 +678,16 @@ namespace Private {
 
   export function CloseButton(props: ICloseButtonProps) {
     return (
-      <button
-        className={`jp-Button jp-mod-minimal ${TOAST_CLOSE_BUTTON_CLASS}${
+      <Button
+        className={`jp-Button jp-icon-hover ${TOAST_CLOSE_BUTTON_CLASS}${
           props.closeIconMargin ? ` ${TOAST_CLOSE_BUTTON_MARGIN_CLASS}` : ''
         }`}
         title={props.title ?? ''}
         onClick={props.close}
+        minimal
```

Comment:
In the newest version of the toolkit package, I deprecated `minimal` in favor of `scale="xsmall"`:

```suggestion
        scale="xsmall"
```